data/scripts/network/cyclopedia_character.lua

```diff
@@ -46,8 +46,9 @@ local function sendCombatStats(self, msg)
 	end
 
 	msg:addU16(0) -- damage reflection (flat, one value for all combat types)
-	msg:addByte(self:getBlessings())
-	msg:addByte(8) -- blessings count
+
+	msg:addByte(self:getBlessingCount())
+	msg:addByte(SERVER_BLESSINGS_COUNT)
```

Comment:
Can you explain why is the constant =5 when it was previously =8

Reply:
> Can you explain why is the constant =5 when it was previously =8

Previously, the constant was 8 because the system checked all blessings from CIP. However, the current TFS does not include all of these blessings, so the constant was adjusted to 5, as I noted in the constants comments.